networking/synthetic/README.md

```diff
@@ -78,14 +78,16 @@ $  ./onload_install
 ```
 $ python stac-prepare-nodes.py -s https://<api-server-ip>:<port> -n <node-1> <node-2> ... <node-N> -i eth0
 ```
-2. Launch producer and consumer pods. Configuration for tests MUST be hosted on a github repo and its url is passed as an argument to the script:
 
+2. Reboot your hosts (just once, for first time)
```

Comment:
I think it will be better to make it as another stage for preparing nodes instead of a note that someone could miss